### ISSUE TITLE
NTR: Lateral thrust during gait (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7980]
New term request Lateral thrust during gait (segal_220817141420): Brief lateral shift at the head of the fibula and upper tibia during the stance phase of gait

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000140
- label: Lateral thrust
- parents: [Abnormal knee physiology (HP:0034670)](https://hpo.jax.org/app/browse/term/HP:0034670)
- pmid: PMID:23948980|PMID:37266363
- def: Lateral thrust (LT) refers to an abnormal knee joint motion in the early stance phase where the knee joint center moves laterally. Also known as varus thrust, LR is defined as the dynamic worsening or abrupt onset of varus (bow-leg) alignment as the limb accepts weight (stance phase), with a return to less varus and more neutral alignment during lift-off and the non-weight-bearing (swing) phase of gait, varus thrust is a potent mechanical risk factor for medial knee osteoarthritis progression.
- comment: LT visualized during walking is associated with a greater medial knee load and an increased risk of medial knee osteoarthritis (OA) progression.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7980
- synonyms: Varus thrust(NONE)
